src/main/java/thrift/model/transaction/Income.java

```diff
@@ -63,6 +65,18 @@ public boolean isSameTransaction(Transaction otherIncome) {
                 && otherIncome.getValue().equals(getValue());
     }
 
+    public boolean getIsJustUpdated() {
+        return isJustUpdated;
+    }
+
+    public void setIsJustUpdated() {
```

Comment:
Same as above. Is it possible to have just one method `setIsJustUpdated(boolean isJustUpdated)` instead of 2 different methods?